### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by gempain**
Fix a few things in the docs (see #19).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ In our case, when we validated a Post object, we have such array of ValidationEr
     property: "title",
     value: "Hello",
     constraints: {
-        length: "$property must be shorter than 10 characters"
+        length: "title must be shorter than 10 characters"
```
Comment (pleerock) [inline README.md]: why did you replace `$property` ?  `$property` is a special variable

Comment (gempain) [inline README.md]: Sorry ! I didn't know, I through it was a typo :)

Comment (pleerock) [inline README.md]: np 😆 thanks for the PR!

Comment (gempain) [inline README.md]: My pleasure sir :)

**Pull request merged**